src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -363,10 +417,15 @@ All parameters are case-insensitive.</value>
 
     Example: "pwsh -SettingsFile c:\myproject\powershell.config.json"
 
+-SSHServerMode | -sshs
+
+    Used for running PowerShell as an SSH subsystem. It is not intended or
```

Comment:
Should we make it a bit more specific?  Like:

```
Used in sshd_config for running PowerShell as an SSH subsystem.
```

Reply:
fixed